tests/test_news.py

```diff
@@ -57,13 +57,14 @@ def test_date_only(self):
         'action': 'opened',
         'number': 1234,
         'pull_request': {
+            'url': 'https://api.github.com/repos/cpython/python/pulls/1234',
```

Comment:
Probably doesn't really matter, but it's `python/cpython`.

Reply:
I know -- but the existing test files (like test_news) had it like this.  Not sure why...   I thought maybe it was a 'just in case' so nothing happened at github.

Reply:
Or I just screwed up typing it the first time. Don't assume someone else didn't mess up before you. 😉